<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface</class>
 <widget class="QWidget" name="interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1373</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>661</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Dés</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="D1">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="imge.qrc">:/new/images/1.gif</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="D2">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="imge.qrc">:/new/images/1.gif</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="D3">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="imge.qrc">:/new/images/1.gif</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="D4">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="imge.qrc">:/new/images/1.gif</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="D5">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="imge.qrc">:/new/images/1.gif</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="checkD1">
            <property name="text">
             <string>Dé 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkD2">
            <property name="text">
             <string>Dé 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkD3">
            <property name="text">
             <string>Dé 3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkD4">
            <property name="text">
             <string>Dé 4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkD5">
            <property name="text">
             <string>Dé 5</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Relancer">
            <property name="text">
             <string>Relancer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="NouveauTour">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Nouveau Tour</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>321</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Feuille de score</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="score">
      <row>
       <property name="text">
        <string>JOUEUR</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>21</red>
         <green>102</green>
         <blue>207</blue>
        </color>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Total de 1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Total de 2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Total de 3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Total de 4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Total de 5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Total de 6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Total</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Si total I&gt;63 alors bonus de 35 points</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Total partie intermédiaire</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Brelan (Total des 3 dés + 20 points)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Carré (Total des 4 dés + 30 points)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Full (Total des 5 dés + 25 points)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Petite suite (30 points)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Grande suite (40 points)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Yams (Total des 5 dés +50 points)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Chance (Total des 5 dés)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Total 2</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>TOTAL</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="8" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="9" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </item>
      <item row="11" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="12" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="13" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="14" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="15" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="16" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="17" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="18" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </item>
      <item row="19" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>91</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>Choix</string>
   </property>
   <widget class="QPushButton" name="tot1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NbeDe1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tot2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tot3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tot5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brelan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="full">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carre">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tot6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tot4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gds">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yams">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Texte">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="locale">
    <locale language="French" country="France"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imge.qrc"/>
 </resources>
 <connections/>
</ui>
